shopfloor/models/shopfloor_menu.py

```diff
@@ -196,6 +196,17 @@ class ShopfloorMenu(models.Model):
         compute="_compute_auto_post_line_is_possible"
     )
 
+    package_process_type_is_possible = fields.Boolean(
+        compute="_compute_package_process_type_is_possible"
+    )
+    package_process_type = fields.Selection(
+        [
+            ("without_package", "Without Package"),
+            ("with_package", "With Package"),
+        ],
+        "Package Proccess Type",
+    )
```

Comment:
It would be nice to move the configuration to the picking type in a separate module e.g. `stock_picking_putinpack_restriction` in `stock-logistics-workflow` and handle the "put in pack" button according to the chosen option. Something similar to https://github.com/OCA/stock-logistics-warehouse/blob/14.0/stock_location_package_restriction/models/stock_location.py#L14